packages/hawtio/src/plugins/camel/route-diagram/routeDiagram.css

```diff
@@ -32,7 +32,12 @@
 .react-flow__node-camel .number {
   align-items: center;
 }
-.react-flow__node-camel .camelNodeLabel {
+.nodeTooltip {
```

Comment:
Using camel case for css names is rare. What about using names with hyphens?  i.e. `node-tooltip`